bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/handler/IPBridgeHandler.java

```diff
@@ -320,8 +320,12 @@ private LutronHandler findThingHandler(int integrationId) {
             if (thing.getHandler() instanceof LutronHandler) {
                 LutronHandler handler = (LutronHandler) thing.getHandler();
 
-                if (handler != null && handler.getIntegrationId() == integrationId) {
-                    return handler;
+                try {
+                    if (handler != null && handler.getIntegrationId() == integrationId) {
+                        return handler;
+                    }
+                } catch (IllegalStateException e) {
+                    logger.trace("Handler for id {} not initialized", integrationId);
```

Comment:
Is this really the only possible cause for this exception? Is trace maybe a little bit too invisible for catching any exception, shall we make it a debug?

Reply:
Hi. It's been several months since I originally made this change.  But as I recall, there was a race condition where a device handler could be created but not yet initialized, and the bridge handler was in some rare cases getting exceptions when it stepped through trying to match IDs from incoming messages to the appropriate handler.  Actually, I still have to go through and make a corresponding change to getIntegrationId() in some other handlers, so thanks for reminding me of it. :-)
As for your questions... Yes, it is the only possible cause of that exception here. And TRACE level is probably ok, since the error should be transient and nobody would normally care about it unless they were trying to tack down some weird initialization problem.

Reply:
Actually, I was wrong. I checked, and getIntegrationId() doesn't need to be updated in any other handlers to fix this problem.